### PR TITLE
Updated matching

### PR DESCRIPTION
**Pull request by RainaAudron**
Updated Glyph.C and Physics.C

https://decomp.me/scratch/RC5Ol Glyph_EndFX 97%
https://decomp.me/scratch/FrlQz GlyphTrigger 98.57%
https://decomp.me/scratch/W6E6k MANA_Pickup 98.57%
https://decomp.me/scratch/Z8Una HUD_Init 96.07% - IDA code was not recognized so had to change the target dli     $v0, 0xFFFFFFFFFFFFFC18 to li     $v0, 0xFFFFFC18 and then it worked.
https://decomp.me/scratch/LlA6d HUD_Setup_Chit_Count 99.29%
https://decomp.me/scratch/EBAAK PhysicsCheckLOS 99.58% 
https://decomp.me/scratch/2CuVK PHYSICS_CheckLineInWorldMask 99.31%

### CODE REVIEW

File: KAIN2/Game/PHYSICS.C [hunk 1]
```diff
@@ -945,17 +945,17 @@ void PhysicsForceSetWater(struct _Instance *instance, int *Time, int Depth, int
...
 
 	struct _PCollideInfo CInfo;
 
 	CInfo.oldPoint = (SVECTOR*)(Data + 8);
-
+	CInfo.newPoint = (SVECTOR*)Data;
 	PHYSICS_CheckLineInWorld(instance, &CInfo);
 
-	return CInfo.type < 1;
+	return (unsigned int)(CInfo.type == 0);
```
Comment (Gh0stBlade) [inline KAIN2/Game/PHYSICS.C:958]: Cast needs removing here then it's good to go.

**Review by Gh0stBlade — CHANGES_REQUESTED** [review]

File: KAIN2/Game/GLYPH.C [hunk 2]
```diff
@@ -1044,7 +1044,7 @@ void HUD_Damp(short* val, short target, short* vel, short max)
 	CriticalDampValue(1, val, target, vel, &accl, max);
 }
 
-void HUD_Init()
+void HUD_Init() //Matching 96.07%
```
Comment (Gh0stBlade) [inline KAIN2/Game/GLYPH.C:1047]: Missing `-`

Comment (RainaAudron) [reply KAIN2/Game/GLYPH.C]: Thanks for spotting that, fixed!

File: KAIN2/Game/PHYSICS.C (hunk 1 above)
Comment (RainaAudron) [reply KAIN2/Game/PHYSICS.C]: Changed it to return CInfo.type == 0; Looks like it builds now.

Comment (Gh0stBlade) [reply KAIN2/Game/PHYSICS.C]: Looks great.

**Review by Gh0stBlade — APPROVED** [review]

**Pull request merged**